sdk/storage/azblob/CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Release History
 
-## 0.5.0 (Unreleased)
+## 0.5.0 (Beta)
 
 ### Breaking Changes
 
 * Complete architectural change for better user experience. Please see the example file for more information.
```

Comment:
I think this should say a bit more about the refactoring.  Some of this has been captured in the README [here](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azblob#readme) so we could link to that part too instead of repeating it.